docs/source/documentation/frontend/index.rst

```diff
@@ -0,0 +1,29 @@
+********
+Frontend
+********
+
+The frontend allows you to monitor your assets (compute plans, tasks, datasets, functions) easily through a user interface.
+
+.. _frontend_kubernetes_pods:
+
+Kubernetes pods
+===============
+
+frontend
+    A single pod managing the frontend. 
+
+.. _frontend_communication:
+
+Communication
+=============
+
+The Backend should be able to reach its Backend through the REST API.
+The access to the API is secured through the use of Json Web Tokens (JWT), which are stored through cookies. Each backend pods has its own tokens, so when working with different backends or restarting pods, it might be necessary to delete related cookies (namely signature, refresh and header.payload) so a new JWT can be created. Otherwise this could block you from logging into the Frontend.   
```

Comment:
```suggestion
The access to the API is secured through the use of Json Web Tokens (JWT), which are stored through cookies. Each backend pods has its own tokens, so when working with different backends or restarting pods, it might be necessary to delete related cookies (namely signature, refresh and header.payload) so a new JWT can be created. Otherwise this could block you from logging into the frontend.   
```